protos/cerbos/cloud/epdp/v1/epdp.proto

```diff
@@ -0,0 +1,18 @@
+// Copyright 2021-2024 Zenauth Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package cerbos.cloud.epdp.v1;
+
+import "cerbos/policy/v1/policy.proto";
+
+option go_package = "github.com/cerbos/cloud-api/genpb/cerbos/cloud/epdp/v1;epdpv1";
+
+message Metadata {
+  string version = 1;
+  repeated string policies = 2;
```

Comment:
Do we need this field? Isn't this information already contained in `source_attributes`?

Reply:
It's for backward compatibility. The first three fields are present in the currently exported metadata object.

Reply:
Let's add a `[deprecated = true]` annotation so that we can stop populating it later.